SirMorland/huld-hub
# Feat/api email domain setup

## Description

Use database to store allowed email domain

## Considerations and implementation

- Add email-domain endpoints
- Use stored email domains in registration endpoint
- Add default email domains
- Set access auth to admins only
### How to test

`cd backend/app && npm run test` or manually test with an invalid domain

Alternatively,
1. Try registering a user with a domain not in the email domain collection, it should not work and you will get and error.
2. Add the email domain
3. Try registering again, it should work this time.
### Test(s) added 
1 test added, 1 test modified


### Screenshots

| Before | After |
| ------ | ----- |
| drag pic here | drag pic here |
